tests/AccountTest.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+use Laracasts\TestDummy\Factory;
+
+class AccountTest extends IntegrationTestCase
+{
+    /**
+     * @test
+     */
+    public function it_deletes_the_user_account()
+    {
+        $user = Factory::create('user');
+        $this->actingAs($user)
+             ->visit('account/delete')
+             ->press('Yes')
+             ->seePageIs('/')
+             ->see('Successfully deleted account.');
+
+        $this->dontSeeInDatabase('users', [
+            'email' => $user->email,
+        ]);
```

Comment:
We'll probably want to test the cascade as well. but this is a great start--thanks!